cluster-scope/components/project-admin-rolebindings/rh-dus/rbac.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: namespace-admin-rh-dus
```

Comment:
I guess here we need a namespace right?

Reply:
Don't think so but I am not sure, as that would make it different from how we do role-bindings in most onboard requests, [see example](https://github.com/operate-first/apps/pull/1559/files#diff-7a6d28a664cfc05e9fa830e17bb41cf9225bcf743fd0754ee9636015989d7a2b).

Reply:
Perfect, we are both right. The namespace inherited from kustomize: https://github.com/operate-first/apps/pull/1706/files#diff-921a80e29e84b44a6dc981a45278ad2ef0dd8c521bf0ae478fd8ea1f62a0af72R3